quickwit/quickwit-doc-mapper/src/default_doc_mapper/field_mapping_entry.rs

```diff
@@ -59,11 +59,12 @@ pub struct FieldMappingEntry {
 /// We do not rely on enum with inline tagging and flatten because
```

Comment:
We should move this docs comment `///` into code comment `//`, it's currently visible in the OpenAPI docs.